content/guides/integrations/sql/go_sql.md

```diff
@@ -0,0 +1,237 @@
+---
+title: "Go"
```

Comment:
nit: s/Go/GoSQL

Reply:
Hmm, I think keeping it as Go suffices because if we add other guides here for Go, they'll fall under "Go/<guide_names...>"